tdrs-frontend/docker-compose.local.yml

```diff
@@ -1,14 +1,15 @@
 # Local version of Docker compose, using the localdev docker build target
 # instead and additionally mounts node_modules for faster builds
-version: "3.9"
+version: "3.4"
 
 services:
   tdp-frontend:
     stdin_open: true # docker run -i
     tty: true        # docker run -t
     build:
       context: .
-      target: nginx
```

Comment:
I think we copied this from the regular docker-compose.yml which resulted in the loop.